Gems/Atom/Asset/ImageProcessingAtom/Assets/Config/ImageBuilder.settings

```diff
@@ -173,17 +173,17 @@
             "_opacity": [ "Opacity" ],
             "_sss": [ "Opacity" ],
             "_trans": [ "Opacity" ],
-			"_transmission": [ "Opacity" ],
-			"_Transmissive": [ "Opacity" ],
-            
+                "_transmission": [ "Opacity" ],
+                "_Transmissive": [ "Opacity" ],
+
             //Reference
             "_ref": [ "ReferenceImage" ],
             "_refhdr": [ "ReferenceImage_HDRLinear" ],
             "_refhdru": [ "ReferenceImage_HDRLinearUncompressed" ],
             "_reflinear": [ "ReferenceImage_Linear" ],
-            
+
             // Reflectance, linear single channel
-			"_cavity": [ "Reflectance" ],
+                "_cavity": [ "Reflectance" ],
```

Comment:
Not sure if it's just how it's being rendered in github diff but some of the spacing here looks off